<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-5H9HEO78" failures="1" tests="5" name="Personal Loan" time="0.09" errors="0" timestamp="2024-08-27T16:47:38 IST">
  <testcase classname="test.BasicDay1" name="Demo" time="0.015">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.BasicDay1.Demo(BasicDay1.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="test.Day2" name="Demo3" time="0.003"/>
  <testcase classname="test.Day4" name="LoginApiHomeLoan" time="0.004"/>
  <testcase classname="test.Day4" name="MobileLoginHomeLoan" time="0.002"/>
  <testcase classname="test.Day4" name="WebLoginHomeLoan" time="0.002"/>
  <testcase classname="test.BasicDay1" name="secondTest" time="0.0">
    <ignored/>
  </testcase> <!-- secondTest -->
</testsuite> <!-- Personal Loan -->
